code/area.dm

```diff
@@ -1387,6 +1387,13 @@ TYPEINFO(/area/diner)
 	requires_power = FALSE
 #endif
 
+/area/watchful_eye_sensor
+	name = "Watchful Eye Sensor Satellite"
+	icon_state = "red"
+	requires_power = FALSE
+	// sound_environment = 12 // what the hell is a sound environment
```

Comment:
?

Reply:
That was meant to be a note to myself to see if i could get the environment to hum or something. Leftover from copy pasting another area for editing. I've removed that line now.